marketplace/charts/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM gcr.io/cloud-builders/gcloud-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        git \
+        wget \
+     && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /bin/helm-downloaded \
+     && wget -q -O /bin/helm-downloaded/helm.tar.gz \
+        https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz \
+     && tar -zxvf /bin/helm-downloaded/helm.tar.gz -C /bin/helm-downloaded \
+     && mv /bin/helm-downloaded/linux-amd64/helm /bin/ \
+     && rm -rf /bin/helm-downloaded \
+     && helm init --client-only \
+     && helm plugin install https://github.com/rimusz/helm-tiller \
+     && helm repo add cloud-marketplace-tools https://cloud-marketplace-tools-charts.storage.googleapis.com
+
+ARG CHART_BUCKET
+ENV CHART_BUCKET $CHART_BUCKET
+
+ARG VERSION
+ENV VERSION $VERSION
+
+COPY . /charts
+
+RUN find /charts \
```

Comment:
I have mixed feelings about this. Why don't we have this as a script inside `dev`, and just run it to push a new version of the chart?

Note that we shouldn't build charts on every commit, since we need to make a versioned tgz file. Ideally, we should only build charts on new release tags. Would suggest moving all building and releasing charts to a follow up.

Reply:
Let's continue this discussion in the follow up pull request. :)